### DISCUSSION TITLE
[ FF7 ] Janky analog movement in latest Canary 1.15.0.12 after #523 enhancement

### DISCUSSION THREAD

**Discussion by shigutso** [julianxhokaxhiu/FFNx#538] [Q&A]
The analog movement seems a bit janky with the latest Canary version 1.15.0.12 installed.
I had to revert to the stable release 1.15.0.0 to get smooth analog movement again.
I tried disabling all mods and using different graphics APIs (Vulkan, DirectX), but nothing worked until I reverted to 1.15.0.0.

This seems to be related to https://github.com/julianxhokaxhiu/FFNx/issues/523

Anyone else having the same issue?
Thanks

**Comment by julianxhokaxhiu** [comment]
Can you please describe janky?

**Reply by shigutso** [reply]
I could say it's the opposite of smooth. It seems like Cloud is snapping into different angles when you turn the analog stick instead of smoothly turning, but it's not also not like his movement is limited to up/down/left/right/diagonals, so the jankyness

**Reply by julianxhokaxhiu** [reply]
This is because this game was never made for Analog and the fact you can use it, was a personal choice I decided to add. Since although I was requested to add the run on tilt, then I decided to lower down a bit the detection threshold so you can walk and run now, depending on how much you tilt. If you want smoother animations feel free to enable the [relative flag](https://github.com/julianxhokaxhiu/FFNx/blob/master/misc/FFNx.toml#L281) which was done exactly for this purpose.

**Reply by shigutso** [reply]
Thanks for your reply! Unfortunately the issue isn't that it feels the movement went back to the original, but that Cloud's movement is broken/janky with this new feature. I'm using the Steam Deck (running the game under wine/proton) but because the analog movement is fine on 1.15.0.0 I'm assuming this is not related.

I will try to make a video and post here. Thanks!

**Reply by julianxhokaxhiu** [reply]
Thanks, looking forward to it then.

**Reply by shigutso** [reply]
The first half of the video is using FFNx 1.15.0.0
Second half is 1.15.0.12
Both are using the same settings, but as I said in my first message, I tried Vulkan and DirectX, with and without mods, but got the same result.
I am doing circular movements with the left analog stick while walking and running.
First half I am holding the X button (PS Controller)
Second half it is using the auto run based on the left analog stick position introduced in https://github.com/julianxhokaxhiu/FFNx/issues/523

Let me know if you need more information! Thanks again for taking a look into this!

https://user-images.githubusercontent.com/5687102/218068492-40f1718b-ba6e-4b0c-a9d4-e5f7caa54f09.mp4

**Reply by julianxhokaxhiu** [reply]
Looks like the dead zone for the Steam Deck is bit more sensitive. Ok I'll look into this, thank you
